README.md

```diff
@@ -0,0 +1,90 @@
+<div align="center">
+  <a href="https://g-research.github.io/">
+    <img alt="G-Research" src="https://github.com/G-Research/brand/raw/main/logo/GR-OSS/logo.svg" height="128" />
+  </a>
+   <h1>Open-Source at G-Research</h1>
+   <p>For a prettier and more inviting experience of looking at GR's current open-source projects.</p>
```

Comment:
I'd like to wordsmith this a little.  It's not bad, I just think it could be better.

Reply:
What do you think about the new description?